src/login/tests/LoginPage.test.jsx

```diff
@@ -664,35 +675,40 @@ describe('LoginPage', () => {
     delete window.location;
     window.location = { href: getConfig().BASE_URL.concat(LOGIN_PAGE), search: `?tpa_hint=${ssoProvider.id}` };
 
-    const loginPage = mount(reduxWrapper(<IntlLoginPage {...props} />));
-    expect(loginPage.find('button#other-ways-to-sign-in').text()).toEqual('Show me other ways to sign in');
+    const { container } = render(reduxWrapper(<IntlLoginPage {...props} />));
+    expect(container.querySelector('#other-ways-to-sign-in').textContent).toEqual('Show me other ways to sign in');
   });
 
   // ******** miscellaneous tests ********
 
   it('should send page event when login page is rendered', () => {
-    mount(reduxWrapper(<IntlLoginPage {...props} />));
+    render(reduxWrapper(<IntlLoginPage {...props} />));
     expect(sendPageEvent).toHaveBeenCalledWith('login_and_registration', 'login');
   });
 
   it('tests that form is only scrollable on form submission', () => {
```

Comment:
How this test is checking the form scrolling behavior?

Reply:
I only migrate the code from enzyme to RTL i did not check the behaviour this can be done in test refactoring